man/systemd-firstboot.xml

```diff
@@ -34,12 +34,22 @@
   <refsect1>
     <title>Description</title>
 
-    <para><command>systemd-firstboot</command> initializes basic system settings interactively during the
-    first boot, or non-interactively on an offline system image. The service is started during boot if
-    <varname>ConditionFirstBoot=yes</varname> is met, which essentially means that <filename>/etc/</filename>
-    is unpopulated, see
+    <para>The <command>systemd-firstboot.service</command> is one of the units which are used to initialize
```

Comment:
actually if you say "The" you probably should also say "service"

Reply:
I was thinking about that too. It's awkward either way. I changed it to 'The systemd-firstboot.service unit' now. It means pretty much the same, about avoid the repetition.